catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -311,6 +311,8 @@ private Option initContentDirectoryMonitorConfig() {
         new BundleInfo("commons-collections", "commons-collections"),
         new BundleInfo("org.apache.commons", "commons-collections4"),
         new BundleInfo("commons-configuration", "commons-configuration"),
+        new BundleInfo("org.apache.commons", "commons-configuration2"),
```

Comment:
❓ Does it still need the old version of `commons-configuration` above?

Reply:
Yes.